sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_encode.py

```diff
@@ -327,9 +319,8 @@ def encode_double(
 
 
 def encode_timestamp(
-    output, value, with_constructor=True, **kwargs  # pylint: disable=unused-argument
-):
-    # type: (bytearray, Union[int, datetime], bool, Any) -> None
+    output: bytearray, value: Union[int, datetime], with_constructor: bool = True, **kwargs: Any  # pylint: disable=unused-argument
+) -> None:
     """
```

Comment:
Can we please remove the double casting below and use the same `int()` fix you added to `encode_long`?
Every time I see `cast` I twitch... to see it on the same value twice in two lines makes me sad :D